force-app/main/default/triggers/ContactTrigger.trigger

```diff
@@ -0,0 +1,18 @@
+trigger ContactTrigger on Contact (after insert,after update,after delete,after undelete) {
+    
+    if(trigger.isAfter){
+        if(trigger.isInsert || Trigger.isAfter){
```

Comment:
why are you checking "Trigger.isAfter" again if you already checked

Reply:
Updated